standards/storing-source-code.md

```diff
@@ -27,9 +27,31 @@ email address to your account](https://help.github.com/articles/adding-an-email-
 and maybe [use that for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)).
 Ask your delivery manager to request you being added to the Github organisation.
 
-Repositories should be [clearly named]({{ '/standards/naming-things' | relative_url }}),
-and have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})
-and enough documentation that someone new can get started with the
-project.
-
-Private repositories are not a good way to protect secrets, and should only be used where access to the code might reveal draft policy decisions.  Secrets should be managed at the platform level.
+Private repositories are not a good way to protect secrets, and should only be used where access to the code might reveal draft policy decisions. Secrets should be managed at the platform level.
+
+If a repository becomes redundant and the codebase is no longer in use or required then it should be archived for # months before it is deleted altogether. This should be done following agreement with relevant parties.
+
+Repositories:
+
+|Requirement|Description|Additional Notes
+|:-:| - | - |
+|Should|be [clearly named]({{ '/standards/naming-things' |relative_url }})||
+|Should|have an [appropriate licence]({{ '/standards/licencing-software-or-code' | relative_url }})|a software license tells others what they can and can't do with the source code|
+|Should|be well documented in the README file|there should be enough documentation that someone new can get started with the project|
```

Comment:
[Can we link this to the READEME principle under the "coding-principles" page](https://skillsfundingagency.github.io/das-technical-guidance/development/coding-principles#11-create-a-useful-readme)

Reply:
Added the link in 👍